app/js/main.js

```diff
@@ -749,112 +767,163 @@ function notifyError(msg) {
+ * @param {String|Number} msg Message to display in confirm dialogue.
  */
-// Create menu
-var menuBar = new gui.Menu({ type: 'menubar' });
-
-// Create sub-menus
-var fileMenuItems = new gui.Menu(),
-    editMenuItems = new gui.Menu(),
-    captureMenuItems = new gui.Menu(),
-    helpMenuItems = new gui.Menu();
-
-//File menu items
-fileMenuItems.append(new gui.MenuItem({
-  label: "New project...",
-    icon: "pngicons/file.png",
-  click: function() {
-  },
-    key: "n",
-    modifiers: "ctrl",
-}));
-fileMenuItems.append(new gui.MenuItem({
-  label: "Open project...",
-    icon: "pngicons/import.png",
-  click: function() {
-  },
-    key: "o",
-    modifiers: "ctrl",
-}));
-fileMenuItems.append(new gui.MenuItem({
-  label: "Main Menu",
-    icon: "pngicons/menu.png",
-  click: function() {
-    openIndex();
-  },
-    key: "m",
-    modifiers: "ctrl",
-}));
-
-//Edit menu items
-editMenuItems.append(new gui.MenuItem({
-  label: "Delete last frame",
-    icon: "pngicons/delete.png",
-  click: function() {
-    undoFrame();
-  },
-  key: "z",
-  modifiers: "ctrl",
-}));
-
-//Capture menu items
-captureMenuItems.append(new gui.MenuItem({
-  label: "Capture frame",
-    icon: "pngicons/capture.png",
-  click: function() {
-    takePicture();
-  },
-  key: "c",
-  modifiers: "ctrl",
-}));
-
-//Help menu items
-helpMenuItems.append(new gui.MenuItem({
-  label: "Give feedback",
-    icon: "pngicons/feedback.png",
-  click: function() {
-      gui.Shell.openExternal('https://github.com/BoatsAreRockable/animator/issues');
-  },
-  key: "/",
-  modifiers: "ctrl",
-}));
-
-// Append sub-menus to main menu
-menuBar.append(
-    new gui.MenuItem({
-        label: 'File',
-        submenu: fileMenuItems
-    })
-);
-menuBar.append(
-    new gui.MenuItem({
-        label: 'Edit',
-        submenu: editMenuItems
-    })
-);
-menuBar.append(
-    new gui.MenuItem({
-        label: 'Capture',
-        submenu: captureMenuItems
-    })
-);
-menuBar.append(
-    new gui.MenuItem({
-        label: 'Help',
-        submenu: helpMenuItems
-    })
-);
-
-// Append main menu to Window
-gui.Window.get().menu = menuBar;
-
-// Create Mac menu
-if (process.platform === "darwin") {
-    menuBar.createMacBuiltin('Boats Animator');
+function confirmSet(func, args, msg) {
+    "use strict";
+    confirmText.innerHTML = msg;
+    confirmContainer.classList.remove("hidden");
+        
+    // Listen if "OK" is pressed
+    confirmOK.addEventListener("click", function() {
+        if (args === undefined) {
+            func();
+        } else {
+            func(args);
+        }
+        confirmContainer.classList.add("hidden");
+    });
+    
+     // Listen if "Cancel" is pressed
+    confirmCancel.addEventListener("click", function() {
+        confirmContainer.classList.add("hidden");
+    });
 }
 
 /**
+ * Display top menu
+ */
+function loadMenu() {
+    // Create menu
+    var menuBar = new gui.Menu({ type: 'menubar' });
```

Comment:
Nit, not important right now: it is all subjective, but the majority of the code uses double quotes, so eventually all quotes should be aligned that way (subject to further discussion if you think single quotes should be used instead ;)).